ChangeLog

```diff
@@ -75,6 +75,7 @@ Changes
      been disabled for lack of a sufficiently recent version of GnuTLS on the CI.
    * Ciphersuites based on 3DES now have the lowest priority by default when
      they are enabled.
+   * Fix typo in config-no-entropy.h submitted by Jacob Schloss in #2479
```

Comment:
This entry should be in the`Bugfix` section, and not in the `changes` section

Reply:
oops, moved entry to Bugfix